packages/flutter/lib/src/rendering/proxy_box.dart

```diff
@@ -3276,17 +3276,17 @@ class RenderSemanticsAnnotations extends RenderProxyBox {
     markNeedsSemanticsUpdate();
   }
 
-  /// Sets the [SemanticsNode.sortOrder] to the given value.
+  /// Sets the [SemanticsNode.sortKey] to the given value.
   ///
-  /// This defines how this node will be sorted with the other semantics nodes
+  /// This defines how this node are sorted with the other semantics nodes
```

Comment:
mention that this is only a local sort amongst siblings?

Reply:
Done.